table/pit-cs-tree.go

```diff
@@ -139,16 +139,27 @@ func (p *PitCsTree) RemoveInterest(pitEntry PitEntry) bool {
 // Originally implemented as FindOrInsertPIT()
 func (p *PitCsTree) FindInterestExactMatch(interest *ndn.Interest) PitEntry {
 	node := p.root.findExactMatchEntry(interest.Name())
-	for _, curEntry := range node.pitEntries {
-		if curEntry.CanBePrefix() == interest.CanBePrefix() && curEntry.MustBeFresh() == interest.MustBeFresh() {
-			return curEntry
+	if node != nil {
```

Comment:
I found a bug where if no exact match entry is found, we get a segfault since the `if node != nil` check is not present. It's present for `FindMatchingDataFromCS`, so seems like it should occur here too.